src/installer/installer.go

```diff
@@ -240,14 +240,7 @@ func (i *installer) waitForControlPlane(ctx context.Context, kc k8s_client.K8SCl
 		i.log.Errorf("Timeout waiting for master nodes, %s", err)
 		return err
 	}
-
-	if err := kc.PatchEtcd(); err != nil {
```

Comment:
u're missing the unpatch

Reply:
Removed the uppatch from the controller and published a new image
quay.io/eranco74/assisted-installer:test